Formula/azure-cli.rb

```diff
@@ -20,6 +20,8 @@ class AzureCli < Formula
 
   def install
     rm_rf "bin/windows"
+    inreplace "package.json", "\"easy-table\": \"0.0.1\",",
+              "\"easy-table\": \"https://github.com/eldargab/easy-table/archive/v0.0.1.tar.gz\","
```

Comment:
Looking at https://github.com/eldargab/easy-table/compare/v0.0.1...v0.0.3#diff-7704a7bcd1f965e637e6ad0c6696e9e9 I think this is more save than bumping it to 0.0.3 (the first version on npm without file system permission errors) and hope that nothing will break